src/components/10-atoms/toggle-switch/react/ControlledWithDummyOnChange.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import AXAToggleSwitchReact from './AXAToggleSwitchReact';
+
+export default class example extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      active: true,
+      receivedEvent: '-',
+    };
+  }
+
+  change = ev => {
+    this.setState({ active: ev.target.checked });
```

Comment:
this is somewhat hard to read without a comment like // ev.target.checked expected to be constant true